hail/python/test/hail/expr/test_ndarrays.py

```diff
@@ -311,6 +316,11 @@ def test_ndarray_ops():
         (ncube1 / nrow_vec, cube1 / row_vec),
         (nrow_vec / ncube1, row_vec / cube1))
 
+    # Missingness tests
+    assert hl.eval(hl.null(hl.tndarray(hl.tfloat64, 2)) + hl.null(hl.tndarray(hl.tfloat64, 2))) is None
+    assert hl.eval(hl.null(hl.tndarray(hl.tfloat64, 2)) + hl._ndarray(np.arange(10).reshape(5, 2))) is None
+    assert hl.eval(hl._ndarray(np.arange(10).reshape(5, 2)) + hl.null(hl.tndarray(hl.tfloat64, 2))) is None
```

Comment:
more self-documenting as:
```
null_2 = hl.null(hl.tndarray(hl.tfloat64, 2))
present_2 = hl._ndarray(np.arange(10).reshape(5, 2))

assert hl.eval(null_2 + null_2) is None
assert hl.eval(null_2 + present_2) is None
assert hl.eval(present_2 + null_2) is None
```

Reply:
At this level of readable, we should probably just inline all the other operators too, for completeness.

And in retrospect the `_2` feels unnecessary.